docs/app/partials/layout-children.tmpl.html

```diff
@@ -1,29 +1,32 @@
 <div ng-controller="LayoutCtrl" class="layout-content" ng-cloak>
 
+  <h3>Children within a Layout Container</h3>
+
   <p>
-    To customize the size and position of elements in a layout, use the
-    <code>flex</code>, <code>flex-order</code>, and <code>flex-offset</code> attributes.
+    To customize the size and position of elements in a layout <b>container</b>, use the
+    <code>flex</code>, <code>flex-order</code>, and <code>flex-offset</code> attributes on the container's <u>child</u> elements:
   </p>
 
   <docs-demo demo-title="Flex Attribute" class="small-demo colorNested">
     <demo-file name="index.html">
       <div layout="row">
-        <div flex>
-          [flex]
+        <div flex="20">
+          [flex="20"]
         </div>
-        <div flex>
-          [flex]
+        <div flex="70">
+          [flex="70"]
         </div>
-        <div flex hide-sm>
+        <div flex hide-sm hide-xs>
           [flex]
         </div>
       </div>
     </demo-file>
   </docs-demo>
+
   <p>
     Add the <code>flex</code> attribute to a layout's child element and the element will flex (grow or shrink) to fit
     the area unused by other elements. <code>flex</code> defines how the element will ajust its size with respect to its
-    parent container and the other other elements within the container.
+    <u>parent</u> container and the other other elements within the container.
```

Comment:
Also, everywhere else that I saw you reference parent/child, you use bold, so I would stick with that.